test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestJcmd.java

```diff
@@ -51,12 +53,19 @@
 	private static final String GC_RUN = "GC.run"; //$NON-NLS-1$
 	private static final String GC_CLASS_HISTOGRAM = "GC.class_histogram"; //$NON-NLS-1$
 	private static final String JCMD_COMMAND = "jcmd"; //$NON-NLS-1$
-	private static String[] JCMD_COMMANDS = { GC_CLASS_HISTOGRAM, GC_RUN, THREAD_PRINT, HELP_COMMAND };
+	public static final String DUMP_HEAP = "Dump.heap"; //$NON-NLS-1$
+	public static final String GC_HEAP_DUMP = "GC.heap_dump"; //$NON-NLS-1$
```

Comment:
This define can be moved after `DUMP_SYSTEM` to match the order within `JCMD_COMMANDS`; similarly, `GC_CLASS_HISTOGRAM` should be before `GC_RUN`. Another option is to re-arrange the defines in alphabetical order, and update `JCMD_COMMANDS` in same order as well.